examples/repository_with_combine.rb

```diff
@@ -44,8 +60,13 @@ class Users < ROM::Relation[:http]
     attribute :email, ROM::Types::String
     attribute :phone, ROM::Types::String
     attribute :website, ROM::Types::String
+
+    associations do
+      has_many :posts, view: :for_user, override: true
```

Comment:
Not the prettiest, but I got lost in trying to have a default view... there are some things related to `preload_assoc`, etc that I need to know about in order to create a default association

Reply:
@ianks how about starting with explicit views? I mean, I don't even think it's possible to just come up with default views, no?